extensions/stripe/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "stripe",
+  "title": "Stripe",
+  "description": "This extension allows you to make an invoice, create a payment, add a new product, and start a new subscription.",
+  "icon": "stripe.png",
+  "author": "pradeepb28",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "create-a-new-invoice",
+      "title": "Create a new invoice",
+      "subtitle": "Stripe",
+      "description": "It creates an invoice for your business on your stripe account",
+      "mode": "view"
```

Comment:
You can set all command to mode "no-view" since otherwise Raycast opens a view and shows the loading indicator.